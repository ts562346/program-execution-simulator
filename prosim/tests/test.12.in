test1 10
BLOCK 1
LOOP 4
  DOOP 2
END
DOOP 5
LOOP 3
  BLOCK 3
END
BLOCK 7
HALT

