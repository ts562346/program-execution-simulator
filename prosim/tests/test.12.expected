00000: BLOCK
00001: DOOP
00003: DOOP
00005: DOOP
00007: DOOP
00009: DOOP
00014: BLOCK
00017: BLOCK
00020: BLOCK
00023: BLOCK
00030: HALT

DOOP count : 5
DOOP time  : 13
BLOCK count: 5
BLOCK time : 17
